Make IDL attribute macros a bit more concise.

eefriedman opened the pull request:
Instead of making the getter and setter separate macros, just combine
them into one, which is almost always what we want to do.

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/servo/servo/7900)

<!-- Reviewable:end -->

bors-servo commented:
:umbrella: The latest upstream changes (presumably #7898) made this pull request unmergeable. Please resolve the merge conflicts.

eefriedman committed:
Make IDL attribute macros a bit more concise.

Instead of making the getter and setter separate macros, just combine
them into one, which is almost always what we want to do.

eefriedman commented:
Rebased.

bors-servo commented:
:umbrella: The latest upstream changes (presumably #7960) made this pull request unmergeable. Please resolve the merge conflicts.

eefriedman commented:
At this point, I'd have to redo this from scratch... and I'm not sure it's worthwhile.  Closing.
